sha-crypt/Cargo.toml

```diff
@@ -21,6 +21,7 @@ sha2 = { version = "0.10", default-features = false }
 # optional dependencies
 rand = { version = "0.8", optional = true }
 subtle = { version = ">=2, <2.5", optional = true, default-features = false }
+base64ct = { git = "https://github.com/ibotty/rustcrypto-formats.git", branch = "shacrypt" }
```

Comment:
You do not need `.git` here.

Reply:
This will be gone if and when a new base64ct version is released.  Or do you think this is actively harmful?

Reply:
I meant you should use `https://github.com/ibotty/rustcrypto-formats`, without the `.git` part.

Reply:
Oh, it worked locally so I assumed that was not a problem.  I will change and rebase in a sec.

Reply:
Also, it looks like you created the formats PR against the `use-base64ct` branch, not `shacrypt`.

Reply:
Hm, removing `.git` has not solved the CI issue. I am not sure why it does not work... Maybe try try to use commit hash instead of branch name?